client/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "core-js": "^3.6.5",
+    "invert-color": "^2.0.0",
```

Comment:
There is no need for a specialized invert color library.  tinycolor2 is already a dependency which is very flexible.  It can give the most readable color for example.